docs/sources/configure-client/language-sdks/dotnet.md

```diff
@@ -9,24 +9,53 @@ aliases:
 
 # .NET
 
-## Supported platforms
+Our .NET Profiler is a powerful tool designed to enhance the performance analysis and optimization of .NET applications. It seamlessly integrates with Pyroscope, offering real-time insights into the resource usage and bottlenecks within your .NET codebase. This integration empowers developers to pinpoint inefficiencies, improve application speed, and ensure resource-efficient operation.
 
-| Spy Name     | Type         | Linux | macOS | Windows | Docker |
-|:------------:|:------------:|:-----:|:-----:|:-------:|:------:|
-| dotnetspy    | `embedded`   |   ✅  |       |         |        |
+### Supported Profiling Types
```

Comment:
We try to use sentence case as our style, see https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/
```suggestion
### Supported profiling types
```